config.yaml

```diff
@@ -42,3 +42,25 @@ modules: [
   netcdf/4.7.4,
   openmpi/4.1.0
 ]
+
+# The 'science_configurations' key is an optional key for users to provide their
+# own custom science configurations. To run benchcab with the default set of science
+# configurations, remove this key from the config file.
```

Comment:
I don't like that the user has to do something to run the default. The `bench_example` repo should be set so a user can run the default with minimum changes. Ideally, no changes.

Reply:
Now that we are writing documentation for the config file, I think we can remove the optional keys (`revision`, `patch`, and `science_configurations`) as this will produce the default configurations. However, we should include a comment somewhere in the config file that makes the user aware that optional keys exist by adding a link to the documentation. What do you think?

Reply:
Yes, and have code examples in the documentation so people can copy-paste directly which I believe you have done.